src/static_canvas.class.js

```diff
@@ -1776,7 +1776,13 @@
       this.contextContainer = null;
       // restore canvas style
       this.lowerCanvasEl.classList.remove('lower-canvas');
-      this.lowerCanvasEl.style = this._originalCanvasStyle;
+      // some browsers (IE) don't allow to redefine style
+      try {
+        this.lowerCanvasEl.style = this._originalCanvasStyle;
+      }
+      catch (err) {
+        fabric.util.setStyle(this.lowerCanvasEl, this._originalCanvasStyle);
```

Comment:
if this would work 100% of the time, why not using it as default?
dispose is not a fast operation, even if setStyle is for some reason more complex, is ok.

Reply:
Because `setStyle` is more slow.
I suppose it should be used as a workaround for IE and similar browsers with restrictions.

Reply:
this is a slow method, is fine to be always slow. please use setStyle 100% of the time

Reply:
Updated.